vignettes/basics_old.Rmd

```diff
@@ -0,0 +1,116 @@
+---
```

Comment:
If I understand correctly this is the previous vignette right?

If yes, next time I suggest using `git mv` for renaming the file - that way the diff in Git would show a moved file not an 'added'

Reply:
Yes, sorry for the mess.